Flick/Controllers/FriendsViewController.swift

```diff
@@ -17,9 +17,11 @@ class FriendsViewController: UIViewController {
 
     // MARK: - Private Data Vars
     private var friends: [UserProfile] = []
+    private var isCollaborators: Bool
 
-    init(friends: [UserProfile]) {
+    init(isCollaborators: Bool, friends: [UserProfile]) {
```

Comment:
can we change `friends` to be more descriptive throughout the file since now this works for both friends and collaborators

Reply:
Maybe change the name of this file too